### PR TITLE
Move po4a stuff to its own dir and add info about man page translations

### PR DESCRIPTION
**Pull request by mariobl**
Hello Karel,
as recommended, I've moved the man page translations to po-man. Moreover, I've update the man page example file and added an incomplete de.po for testing purposes.

Best Regards,
Mario

### CODE REVIEW

File: man-common/manpage-stub.adoc [hunk 1]
```diff
@@ -11,36 +25,42 @@ foobar - do nothing
...
+mailto:john.doe@example.com[John Doe]
+// If you need more than one address, use a comma-separated list or leave empty lines between the entries.
 
-[[see_also]]
 == SEE ALSO
 
 *foo*(1),
 *bar*(1)
-
+
```
[the remark this answers is not in the document]
Comment (mariobl) [reply man-common/manpage-stub.adoc]: Good catch, fixed. Thanks for the hint.

**Comment by karelzak** [comment]
I guess I can merge it for -rc2, right?

**Comment by mariobl** [comment]
OK.

**Comment by karelzak** [comment]
Merged. 

For now (and during v2.37) the translations are disabled by default. You need `./configure --enable-poman` to enable. It works for `make`, and `make install` (test it with `DESTDIR=/tmp/something make install`). You can also test it by `make -C po-man` to generate translated .adoc(s) and man pages.

There are two issues:
   * asciidoctor: ERROR: de/blockdev.8.adoc: line 8: non-conforming name section body
   * and po-man/de/libblikid.3.adoc does not generate any man page.

The remaining thing is that it installs all man pages, including man pages for disabled tools. I'll fix it later.

Anyway, it looks good.

**Comment by mariobl** [comment]
> There are two issues:
> 
> * asciidoctor: ERROR: de/blockdev.8.adoc: line 8: non-conforming name section body
> * and po-man/de/libblikid.3.adoc does not generate any man page.
>
Fixed; see my latest pull request. Don't bother with the commit in hardlink.c, this has been fixed subsequently by the upstream merge.

> The remaining thing is that it installs all man pages, including man pages for disabled tools. I'll fix it later.
> 
Do you mean man pages in general, or only translated versions?

**Comment by karelzak** [comment]
> > The remaining thing is that it installs all man pages, including man pages for disabled tools. I'll fix it later.
> 
> Do you mean man pages in general, or only translated versions?

The translated versions, the rest already works as expected.

**Comment by mariobl** [comment]
OK, then it is no problem for the time being. I don't expect that any downstream packager enables the creation of the translated man pages explicitely. Anyway, I will keep an eye on the upcoming util-linux packages, at least for the distributions which have a manpages-l10n package.

**Comment by karelzak** [comment]
Yes, during v2.37 we need to make it more stable, translate it to more languages and integrate it to TP.org. And then in v2.38 we can assume use by downstream maintainers and users. For example for Fedora and RHEL, I'd like to create util-linux sub-packages with translated man pages.

**Pull request merged**